projects/kit/components/carousel/carousel.directive.ts

```diff
@@ -20,7 +22,12 @@ export class TuiCarouselDirective extends Observable<unknown> {
         this.visible$,
     );
 
-    private readonly output$ = combineLatest([this.duration$, this.running$]).pipe(
+    private readonly output$ = combineLatest([
```

Comment:
Should we instead do `isPlatformServer(...) ? EMPTY : ...` maybe, so to minimize effect on overall code?